ai-ml/trainium-inferentia/variables.tf

```diff
@@ -13,7 +13,7 @@ variable "region" {
 
 variable "eks_cluster_version" {
   description = "EKS Cluster version"
-  default     = "1.27"
+  default     = "1.28"
```

Comment:
you can rebase from the main as this code is already in the main

Reply:
Sure, will do.